services/core-web/src/components/Forms/permits/SitePropertiesForm.js

```diff
@@ -0,0 +1,195 @@
+                }
+              />
+            </Col>
+          </Row>
+          <Row gutter={16}>
+            <Col span={24}>
+              <div className="field-title">{isCoalOrMineral ? "Disturbance*" : "Disturbance"}</div>
+              <Field
+                id="mine_disturbance_code"
+                name="mine_disturbance_code"
+                component={RenderMultiSelect}
+                data={
+                  this.props.site_properties?.mine_tenure_type_code
+                    ? this.props.conditionalDisturbanceOptions[
+                        this.props.site_properties?.mine_tenure_type_code
+                      ]
+                    : null
+                }
+                validate={isCoalOrMineral ? [required] : []}
+              />
+            </Col>
+          </Row>
+        </FormSection>
+        <Row gutter={16}>
+          <Col span={24}>
+            <Field
+              id="exemption_fee_status_code"
+              name="exemption_fee_status_code"
+              label="Inspection Fee Status"
+              placeholder="Inspection Fee Status will be automatically populated."
+              component={RenderSelect}
+              disabled
+              data={this.props.exemptionFeeSatusDropDownOptions}
+            />
+          </Col>
+        </Row>
+        <Row gutter={16}>
+          <Col span={24}>
+            <Field
+              id="exemption_fee_status_note"
+              name="exemption_fee_status_note"
+              label="Fee Exemption Note"
+              component={RenderAutoSizeField}
+              validate={[maxLength(300)]}
+            />
+          </Col>
+        </Row>
+        <div className="right center-mobile">
+          <Popconfirm
+            placement="topRight"
+            title="Are you sure you want to cancel?"
+            okText="Yes"
+            cancelText="No"
+            disabled={this.props.submitting}
+            onConfirm={() => this.props.closeModal()}
+          >
+            <Button className="full-mobile" type="secondary" disabled={this.props.submitting}>
+              Cancel
+            </Button>
+          </Popconfirm>
+          <Button
+            className="full-mobile"
+            type="primary"
+            htmlType="submit"
+            loading={this.props.submitting}
+          >
+            Save
+          </Button>
+        </div>
+      </Form>
+    );
+  }
+}
+
+SitePropertiesForm.propTypes = propTypes;
+const selector = formValueSelector(FORM.EDIT_SITE_PROPERTIES);
+
+export default compose(
+  connect((state) => ({
+    mineTenureTypes: getMineTenureTypeDropdownOptions(state),
+    conditionalCommodityOptions: getConditionalCommodityOptions(state),
+    conditionalDisturbanceOptions: getConditionalDisturbanceOptionsHash(state),
+    site_properties: selector(state, "site_properties"),
+    exemptionFeeSatusDropDownOptions: getExemptionFeeSatusDropDownOptions(state),
+  })),
+  reduxForm({
+    form: FORM.EDIT_SITE_PROPERTIES,
```

Comment:
This doesn't need `enableReinitialize: true` or anything?

Reply:
I'll add it -I didn't notice any negative side effects without it, but it's good to have.